public/css/app.css

```diff
@@ -149,47 +162,47 @@ img.logo {
 }
 
 .banner-content span.breach-info {
-  margin-top: calc(5px + .25vh + .25vw);
-  max-width:300px;
-  width:100%;
-  font-weight:400;
+  margin-top: calc(5px + 0.25vh + 0.25vw);
+  max-width: 300px;
+  width: 100%;
+  font-weight: 400;
 }
 
 #breach-title-date span.breach-info {
-  margin-top: 0px;
+  margin-top: 0;
 }
 
 .banner-content span.breach-info span.medium {
-  display:block;
+  display: block;
 }
 
 #breach-title-wrapper {
-  display:flex;
-  margin-bottom:calc(.5vh + .5vw);
+  display: flex;
+  margin-bottom: calc(0.5vh + 0.5vw);
 }
 
 #breach-title-date {
   display: flex;
-  align-items:center;
```

Comment:
@lesleyjanenorton I wasn't sure what the right answer for this was. It looks like this selector had two `align-items` values ("center" and "flex-start"). I opted for the latter one, figuring it was probably overriding "center" anyways. But happy to change to a different value in this PR or a new one...